src/yardstick/store/result_set.py

```diff
@@ -19,10 +19,8 @@ def _store_root(store_root: str | None = None):
 
 
 def clear(store_root: str | None = None):
-    try:
+    with contextlib.suppress(FileNotFoundError):
         shutil.rmtree(_store_root(store_root=store_root))
```

Comment:
Does it make more sense to pass `ignore_errors=True` to shutil.rmtree? https://docs.python.org/3/library/shutil.html#shutil.rmtree ? Or does that method sometimes raise other errors that we do want?

Reply:
nope -- you're suggestion is better, I'll incorporate